src/query/plan/operator.cpp

```diff
@@ -5770,17 +5771,20 @@ UniqueCursorPtr HashJoin::MakeCursor(utils::MemoryResource *mem) const {
   return MakeUniqueCursorPtr<HashJoinCursor>(mem, *this, mem);
 }
 
-RollUpApply::RollUpApply(const std::shared_ptr<LogicalOperator> &input,
-                         std::shared_ptr<LogicalOperator> &&second_branch)
-    : input_(input), list_collection_branch_(second_branch) {}
+RollUpApply::RollUpApply(const std::shared_ptr<LogicalOperator> &&input,
```

Comment:
I am pretty confident you need to remove here `const` to capture by rvalue reference to be able to move it afterwards, and then you can std::move value into intput_
```suggestion
RollUpApply::RollUpApply(std::shared_ptr<LogicalOperator> &&input,
```

Reply:
Good catch! Thank you!